src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,15 @@
+import { ObjectLink } from "@/constants/index.types";
+import Card from "./Card/Card";
+import styles from "./CardList.module.scss";
+
+interface CardListProps {
+  links?: ObjectLink[];
+}
+
+export default function CardList({ links }: CardListProps) {
+  return (
+    <section className={styles.CardList}>
+      {links && links.map((link) => <Card link={link} key={link.id} />)}
```

Comment:
이렇게 표현해줘도 괜찮을 것 같네요!
```suggestion
      {links?.map((link) => <Card link={link} key={link.id} />)}
```